include/ddc/chunk_span.hpp

```diff
@@ -11,8 +11,9 @@
 
 #include "ddc/chunk_common.hpp"
 #include "ddc/discrete_domain.hpp"
+#include "ddc/memory.hpp"
 
-template <class, class>
+template <class ElementType, class, class Allocator = HostAllocator<ElementType>>
```

Comment:
Why specify the default allocator here and not in `include/ddc/chunk.hpp` ?

Also, I would rely on `std::allocator` in this CPU-only branch

Reply:
1) In fact I thought it had to be specified on the first declaration but it does not seem to be required. I will change update this.

2) Why using `std::allocator` ? We were not relying on this allocator before.